honeybee_energy/run.py

```diff
@@ -215,7 +313,31 @@ def prepare_idf_for_simulation(idf_file_path, epw_file_path):
     return directory
 
 
-def _output_files_from_directory(directory):
+def _prepare_osw(osw_json):
```

Comment:
This is more like checking for osw. prepare_osw sounds like like a method that you prepare the file itself.

Reply:
Ok. Will  change it to `check_osw`.

Reply:
Done.